.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Add `.prettierignore` and `.eslintignore` files, add all build directory, all file you don't want commit.  

```
node_modules
.clever.json
apps/api/admin
```

There files improve the watcher performance of vscode or other IDE